package.json

```diff
@@ -3,17 +3,26 @@
     "start": "nf start",
     "clean": "rm -rf ./dist && mkdir ./dist",
     "build": "yarn run clean && yarn run build-server && yarn run build-client",
+    "stylelint": "stylelint \"src/client/styles/**/*.scss\" --syntax scss --fix",
     "start-server": "nodemon --watch ./src/server --watch ./src/shared --exec \" node src/server/\"",
     "start-client": "webpack-dev-server --host 0.0.0.0 --hot --config webpack.dev.js",
     "build-server": "yarn run babel-server && yarn run babel-shared",
     "build-client": "BABEL_ENV=production webpack -p --config webpack.prod.js && rsync -r ./static/ ./dist/static/",
     "babel-server": "BABEL_ENV=production babel ./src/server --out-dir ./dist/server",
-    "babel-shared": "BABEL_ENV=production babel ./src/shared --out-dir ./dist/shared"
+    "babel-shared": "BABEL_ENV=production babel ./src/shared --out-dir ./dist/shared",
+    "precommit": "lint-staged"
   },
   "name": "senseis-top",
   "version": "0.0.1",
   "license": "MIT",
   "author": "Alex Zaworski",
+  "lint-staged": {
+    "src/client/styles/**/*.scss": [
+      "stylelint --fix --syntax scss",
+      "git add"
+    ]
+  },
+  "browserslist": ["> 1%"],
```

Comment:
I'm comfortable being narrower here, something like: `> 0.5%, last 1 version, not dead` is fine with me.